docs/admin/access.rst

```diff
@@ -53,9 +53,14 @@ Adding permissions to languages, components or projects
 
 .. note::
 
-    This feature is not available for projects running the Hosted Libre plan.
+    This feature is unavailable for projects using the Libre plan on Hosted Weblate.
 
-You can additionally grant permissions to any user based on project, component or language
+Members are granted any permissions assigned to groups they are in, so you can
+grant the user multiple permissions at once. Create groups and attach them to a project,
+component, or language. You can put users in multiple groups, and permissions
+can overlap between them.
+
+Granting any selected permissions based on project, component or language
```

Comment:
@Fat-Zer commented this on https://hosted.weblate.org/translate/weblate/documentation/en/?checksum=d95ecb078debd2ce#comments: 

> "_Granting any selected permissions_" — sounds incorrect... Probably it should be reverted to "_You can additionally grant permissions to any user_"... or something like "_Granting any selected permissions ... **you can achieve world domination**_"... or it might supposed to be imperative: "_**Grant** any..._"